lib/matplotlib/backends/backend_wx.py

```diff
@@ -951,24 +951,13 @@ def _on_close(self, event):
         self.figmgr.frame = None
         # remove figure manager from Gcf.figs
         Gcf.destroy(self.figmgr)
+        try:  # See issue 2941338.
```

Comment:
Where is issue 2941338? I can't find it anywhere and googling just leads me back to this code block.

Reply:
Probably that was on the old sourceforge tracker.  It's probably not really useful anymore (I can't find it either), but I didn't want to bother removing it.  I can, though, if you prefer.

Reply:
I'm indifferent... I usually agree with leaving comments in the code, but when they lead to nowhere it can also lead to confusion.

This one needs a rebase too.

Reply:
left it for now, and rebased.